test/test_cli.py

```diff
@@ -35,11 +35,25 @@ def test_cli_shows_help_with_long_help_flag(self, cli_invocation, output, client
 
     @cli_test(["-v"])
     def test_cli_shows_version_with_short_version_flag(self, cli_invocation, output, client):
-        self.assertRegex(str(output), f"Conjur Python CLI version {(__version__)}")
+        self.assertRegex(str(output), f"Conjur CLI version {(__version__)}")
 
     @cli_test(["--version"])
     def test_cli_shows_version_with_long_version_flag(self, cli_invocation, output, client):
-        self.assertRegex(str(output), f"Conjur Python CLI version {format(__version__)}")
+        self.assertRegex(str(output), f"Conjur CLI version {format(__version__)}")
+
+    @cli_test(["-v"])
+    def test_cli_check_copyright_short_version_flag(self, cli_invocation, output, client):
+        self.assertRegex(str(output), '''
+Copyright © 2020 CyberArk Software Ltd. All rights reserved.
```

Comment:
we need to remove the © because this was leading to failures in rhel and windows

Reply:
This should be fine as-is as long as the file has the `# -*- coding: utf-8 -*-` at the top

Reply:
The copyright was found on the top of `cli.py` which has the `# -*- coding: utf-8 -*-`. When we packed the executable on a RHEL/Windows machine and ran it on a RHEL/Windows machine we were getting the following error:

<img width="1003" alt="Screen Shot 2020-12-07 at 17 47 29" src="https://user-images.githubusercontent.com/19418506/102178473-20c5d600-3eae-11eb-8724-1bb8573cb503.png">

xa9 is the [copyright](https://www.codetable.net/hex/a9) so I thought best to remove it since we say Copyright anyways